src/components/Logout.tsx

```diff
@@ -0,0 +1,23 @@
+import { useAuth } from "../context/useAuth";
+
+type LogoutProps = {
+  setUserMenu: (arg: boolean) => void;
```

Comment:
I would change this to a more generic function. For example onLogin, here we can pass a callback that contains more logic that needs to be run when the user is logged in. Now we pass setUserMenu that doesn't really have to do with the login function.